atlas/AtlasMl/atlasml/clients/weaviate.py

```
@@ -104,56 +125,66 @@ def close(self):
         return uuid
-    
+
     def get_embeddings(self, collection_name: str, id: str):
         """Get embeddings for a given ID from the specified collection."""
-        logger.info(f"--- GETTING EMBEDDINGS FROM WEAVIATE COLLECTION '{collection_name}' ---")
+        logger.info(
+            f"--- GETTING EMBEDDINGS FROM WEAVIATE COLLECTION '{collection_name}' ---"
+        )
         self._check_if_collection_exists(collection_name)
 
         embedding = self.get_embeddings_rest(collection_name, id)
```

Pylint (3.3.7)

[error] Instance of 'WeaviateClient' has no 'get_embeddings_rest' member

(E1101)